src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -7,40 +7,64 @@ category: adapter
 i18nReady: true
 ---
 
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
 An SSR adapter for use with Cloudflare Pages Functions targets. Write your code in Astro/Javascript and deploy to Cloudflare Pages.
 
 ## Install
 
 Add the Cloudflare adapter to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
```

Comment:
[suggestion]: Since there's two methods to installing the adapter, I think it would be good idea to separate them into sub sections and hyperlink each one. Something like this: 
##Install
Now there's two ways you can go about installing the Cloudfare adapter. Click on the path you want to use:
1. [`astro add` command](link to command section)
2. [Manual] (link to manual)
### `astro add` command
### Manual
Doing this could make the user experience more personalized.

Reply:
Voxel is addressing this!